front/app/containers/EventsShowPage/components/MetadataInformation/EventDateStylized/index.tsx

```diff
@@ -18,14 +16,14 @@ const EventDateStylized = ({ event }: Props) => {
   const theme = useTheme();
   const startAtMoment = moment(event.attributes.start_at);
   const endAtMoment = moment(event.attributes.end_at);
-  const startDateMonth = toThreeLetterMonth(startAtMoment.toISOString(), 'month');
-  const endDateMonth = toThreeLetterMonth(endAtMoment.toISOString(), 'month');
+  const startDateMonth = startAtMoment.format('MMM');
+  const endDateMonth = endAtMoment.format('MMM');
   const isEventMultipleDays =
```

Comment:
The "moment" is already linked to the correct timezone. Using that toThreeLetterMonth function was converting it back to utc.